editor/editor_node.cpp

```diff
@@ -5024,6 +5024,35 @@ bool EditorNode::immediate_confirmation_dialog(const String &p_text, const Strin
 	return singleton->immediate_dialog_confirmed;
 }
 
+Color EditorNode::peek_cfg_theme_color() {
+	// Default black(0,0,0,1)
+	Color result;
+
+	/*
+	 * This static method is used during the initialization phase of the initial window,
+	 * care should be taken to detect whether each instance of the class is available.
+	 */
+	EditorPaths *p_editorPaths = EditorPaths::get_singleton();
```

Comment:
```suggestion
	EditorPaths *editor_paths = EditorPaths::get_singleton();
```
We use `p_` as prefix for function parameters, not for pointers. And we use snake case for identifiers.